src/components/ComboButton/ComboButton.js

```diff
@@ -55,9 +58,10 @@ const ComboButton = ({ children, className, direction }) => {
       <Button
         className={classnames(className, `${namespace}--primary`)}
         disabled={disabled}
+        href={href}
         iconDescription={iconDescription}
         kind="primary"
-        key={button.id}
+        key={children}
```

Comment:
Would you mind explaining the reasoning behind this and https://github.com/carbon-design-system/ibm-security/pull/112/files#diff-3773b98c8952dbe82cae025bb1042247R113 ?

As React recommends keys to be strings ( https://reactjs.org/docs/lists-and-keys.html#keys ), we can't be entirely certain that the `children` for these will always be as such.

Reply:
`children` at this point is often a string, which is why I changed it from `button.id` (and similarly for other instances I changed, because errors where still being shown for no `key` n the console). But it's possible that `children` could be something other than a string so I'm just going to remove these changes from the PR so we can think about it more in a separate set of changes 👍